source/tests/system/nisync_driver_api_tests.cpp

```diff
@@ -1,20 +1,43 @@
 #include <gtest/gtest.h>
 
 #include "device_server.h"
-#include "nisync/nisync_service.h"
 #include "enumerate_devices.h"
+#include "ivi.h"
+#include "nisync/nisync_client.h"
 
 namespace ni {
 namespace tests {
 namespace system {
 
 namespace nisync = nisync_grpc;
 
-static const ViStatus kSyncInvalidSrcTerminal = 0xBFFA4032;
-static const char* kTestSessionName = "TestSession";
-static const char* kEmptySessionName = "";
-static const char* kInvalidRsrcName = "InvalidName";
-static const char* kInvalidTerminal = "Invalid";
+constexpr auto INVALID_SOURCE_TERMINAL = 0xBFFA4032;
+constexpr auto SESSION_NAME = "TestSession";
+constexpr auto EMPTY_SESSION_NAME = "";
+constexpr auto INVALID_RESOURCE_NAME = "InvalidName";
+constexpr auto INVALID_TERMINAL = "Invalid";
+constexpr auto NISYNC_VAL_OSCILLATOR = "Oscillator";
+constexpr auto NISYNC_VAL_CLKOUT = "ClkOut";
+constexpr auto NISYNC_ERROR_SRC_TERMINAL_INVALID = 0xBFFA4032;
+constexpr auto NISYNC_VAL_CLKIN = "ClkIn";
+constexpr auto NISYNC_VAL_SWTRIG_GLOBAL = "GlobalSorftwareTrigger";
+constexpr auto NISYNC_VAL_PXIEDSTARC = "PXIe_DStarC";
+constexpr auto NISYNC_VAL_SYNC_CLK_FULLSPEED = "SyncClkFullSpeed";
+constexpr auto NISYNC_VAL_CLK100 = "PXIe_Clk100";
+constexpr auto NISYNC_VAL_PFI0 = "PFI0";
+constexpr auto NISYNC_VAL_PFI1 = "PFI1";
+constexpr auto NISYNC_VAL_SYNC_CLK_ASYNC = "SyncClkAsync";
+constexpr auto NISYNC_VAL_1588_CLK_ACCURACY_WITHIN_1_USEC = 4;
```

Comment:
Note: This one was a mess because most of the constants used were strings and besides that Sync doesn't have any enums in the proto besides the attributes. That being said, there were a few attribute replacements to instead use the attribute enum from the proto file so these changes still seem worth it.